openeo_driver/views.py

```diff
@@ -1187,10 +1188,16 @@ def job_result_item_url(item_id, is11 = False) -> str:
 
 
             if len(result_metadata.items) > 0 :
```

Comment:
```suggestion
            if result_metadata.items:
```

I would take the opportunity to replace this as it it shorter, safer and expresses the intent better.